frontend/src/pages/journal/components/journal-card/styles.module.scss

```diff
@@ -0,0 +1,30 @@
+.item {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  min-width: 317px;
+  min-height: 83px;
+  margin-top: 15px;
+  padding: 0;
+  font-weight: 600;
+  font-size: 16px;
+  font-family: Montserrat, sans-serif;
```

Comment:
We already have default font family in body component, just forget about mentioning it)